todo-list-connected-to-api/maryam/src/TodoItem.js

```diff
@@ -0,0 +1,72 @@
+import React, {useState} from 'react';
+import {
+  ListItem,
+  ListItemText,
+  Checkbox,
+  IconButton,
+  ListItemSecondaryAction
+} from '@material-ui/core';
+import DeleteIcon from '@material-ui/icons/Delete';
+// ----------------------------------------------------------------
+const TodoItem = ({value}) => {
+    const [check, setCheck] = useState(false);
+    const itemClickedHandler = () => {
+        if (value.checked === false) {
+            fetch(`https://todolist.ehsan-rafee.ir/api/todolist/${value.id}`,{
+                method: 'PUT',
+                headers: {
+                    'Content-Type':'application/json'
+                },
+                body: JSON.stringify({
+                    title: value.title,
+                    checked: true
+                })
+            })
+            .then(response => response.json())
+            .then(data => setCheck(true))
+            .catch(error => console.error(error))
+    }
+}
+
+    const deleteClickedHandler = () => {
+        fetch(`https://todolist.ehsan-rafee.ir/api/todolist/${value.id}`,{
```

Comment:
__[HIGH]__  
Same here. You should inform react that an item is deleted using `setToDoItems`, otherwise UI doesn't change.